torchrl/trainers/trainers.py

```diff
@@ -933,12 +932,11 @@ def __init__(
     def __call__(self, batch: TensorDictBase) -> Dict:
         out = None
         if self._count % self.record_interval == 0:
+            torch.cuda.empty_cache()
```

Comment:
I think we added this because of weird cuda mem issues which I believe are solved now. Can we remove this and try to train something like dreamer while looking at the mem on devices? 
Asking this because this operation takes a considerable amount of time :)

Reply:
yes i agree let's remove this